instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import dummyData from "../../dummy-data"
+import PostContainer from "../PostContainer/PostContainer"
+import SearchBar from "../SearchBar/SearchBar"
+
+
+class PostsPage extends Component {
+  constructor(){
+    super()
+    this.state = {
+      posts: dummyData
+      
+    }
+  }
+
+
+  changeHandler = e => {
+    this.setState({input: e.target.value})
+  }
+
+  find = event => {
+    event.preventDefault();
+    const post = this.state.posts.filter(post=>{
+      if(post.username===this.state.input){
+        return true;
+      }
+      else {
+        return false;
+      }
+      
```

Comment:
Right here, instead of using if/else, you can do -> (post => post,username===this.state.input) to shorten down your code.